docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -0,0 +1,81 @@
+
+Getting Started with Galaxy
+===========================
+
+Tutorial Goals
+--------------
+
+-  Learn about `Galaxy <https://training.galaxyproject.org/>`__
+-  Create and run a basic Galaxy workflow
+-  Export the workflow to a file
+-  Setup a GitHub account and repository
+-  Push the workflow to GitHub
+-  Make a GitHub release
+
+Create a basic Galaxy workflow
+------------------------------
+
+Create and run your workflow in Galaxy. Here  is a tutorial for `Creating, Editing, Importing Galaxy Workflows <https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html>`__
+
+Export the workflow to a file
+-----------------------------
+
+- Click on the Galaxy UI Workflow link at the top of the page.
+- Click on the workflow name to expose the drop down menu.
+- Click Download
+
+The exported JSON file with a '.ga' suffix describes the inputs,
+outputs, and Galaxy Tool Shed dependencies for your workflow.
+
+
+.. figure:: /assets/images/docs/galaxy_download.png
```

Comment:
Looks like this is either the wrong path or not committed; I don't see an image at https://docs.dockstore.org/en/feature-galaxy-worflows/getting-started/getting-started-with-galaxy.html